### ISSUE TITLE
ipstack support

### ISSUE THREAD

**Issue by fcordenes** [navinpeiris/geoip#8]
Hi guys,

There is any chance for given support for the ipstack url request?
I see that the current request is considers the structure of freegeoip:

`curl *url*/json/*ip*`

but now the request for ipstack is a little different:

`curl http://api.ipstack.com/*ip*?access_key=*access_key*`

Thanks

**Comment by sitch** [comment]
Anyone working on this?

**Comment by nathany** [comment]
@sitch We need this right away, so I'm working on a PR this week.

**Comment by nathany** [comment]
The new ipstack API seems less than perfect.

The test suite comes back with lovely things like this, with a 200 OK.

```json
{"ip":"invalidhost","country_code":null,"country_name":null,"region_code":null,"region_name":null,"city":null,"zip_code":null,"time_zone":null,"latitude":null,"longitude":null,"metro_code":""}%                                                                                                              
```

From what I gather, that was an error before with freegeoip. And this was tested with legacy mode.

Also, I'm not sure it's worth trying to support the standalone freegeoip code alongside the new ipstack API.

**Comment by acabreragnz** [comment]
If somebody needs a fast solution, replace geoip with this in your mix.exs:

`{:geoip, git: "https://github.com/arbitinc/geoip.git", tag: "0.1.4-apistack-bc"}`

Then in your secret:
```
config :geoip,
			 url: "http://api.ipstack.com/",
			 api_access_key: "your_api_access_key"
```

[Here](https://github.com/arbitinc/geoip/pull/1) you could see the pull request, not a long-term solution, but fix the issue at least for now

**Comment by navinpeiris** [comment]
Sorry for the delay on this guys and thanks for the fast solution @acabreragnz, got side tracked with some urgent issues in my day job.

A new version of geoip with support for self hosted geoip, ipstack and ipinfo are in the works, will try to get that out in the next couple of days.

**Comment by navinpeiris** [comment]
hi all, support for both ipstack and ipinfo was released with version `0.2.0`. This includes some breaking changes, please see the changelog for more details.
